### ISSUE TITLE
learning rate is 0, output dimensions of some conv layers

### ISSUE THREAD

**Issue by andwany** [carykh/alignedCelebFaces#12]
Hi, 
Why is learning rate 0 in ACF_just_encode.py and anyFaceEncoder.py?

Also, there is this line in anyFaceEncoder.py:
conv0 = tf.layers.conv2d(inputs=inputs_, filters=120, kernel_size=(3, 3), padding='same', activation=tf.nn.relu)
\# Now 64x64x25

Why is the output shape 64x64x25? Shouldn't it be 64x64x120?

Thanks

**Comment by carykh** [comment]
oh, the files I used for training are super messy right now, since I really only planned on the public using the final_slider_tool.

The learning rate is supposed to be something like 0.0002, but it was 0 for a time because I was testing how to just decode images by running that code and I didn't want any weights to change when I did that.

Also, don't trust the comments: the parameters are right, it should say 64x64x120. The channel count used to be much lower, but I bumped it up late into the project without fixing the comments!
